client/js/store/modules/game.js

```diff
@@ -183,6 +187,8 @@ export default {
       if (Object.hasOwnProperty.call(newGame, 'oneOffTarget'))
         state.oneOffTarget = _.cloneDeep(newGame.oneOffTarget);
       else state.oneOffTarget = null;
+
+      if (Object.hasOwnProperty.call(newGame, 'isRanked')) state.isRanked = newGame.isRanked;
```

Comment:
Boy do I hate the way we do this a lot. I know why now and I don't blame you for it at all, but it's highly confusing the have the same function fire with both a full game object and a limited game object. I would love to align on a singular schema for the game object that comes through and make sure it has all the properties each time so we don't need to be this defensive.

Reply:
Yes I think that's completely doable. Worth an issue

Reply:
https://github.com/cuttle-cards/cuttle/issues/126